modules/dataframeqt/include/inviwo/dataframeqt/processors/dataframetable.h

```diff
@@ -93,12 +90,9 @@ class IVW_MODULE_DATAFRAMEQT_API DataFrameTable : public Processor,
     IntVec2Property position_;
     BoolProperty showIndexColumn_;
     BoolProperty showCategoryIndices_;
-    BoolProperty vectorCompAsColumn_;
+    BoolProperty showFilteredRowCols_;
```

Comment:
might consider filtering of columns too in the future I think

Reply:
true, in case of the TableView that would just be hiding/showing arbitrary columns.